base_user_role_history/models/base_user_role_line_history.py

```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
-# Copyright 2019 ACSONE SA/NV
+# Copyright 2022 ACSONE SA/NV
```

Comment:
Please keep 2019 (or put 2019-2022, but that is not necessary)

Reply:
Changes are made